emsize.py

```diff
@@ -37,32 +37,47 @@ def error(text):
 
 
 def parse_args(argv):
-  parser = argparse.ArgumentParser(description=__doc__)
-  parser.add_argument('-format', '--format')
-  parser.add_argument('file')
-  args = parser.parse_args(argv)
-  return args.file
+  files = []
+  params = []
+  appendFormat = True
+  ret = 0
+  for arg in argv:
+    if (arg.startswith('-')):
+      params.append(arg)
+    else:
+      files.append(arg)
+    if (arg.startswith('--format')):
+      appendFormat = False
+  if appendFormat:
+    params.append('--format=sysv')
+  for file in files:
+    ret = print_sizes(file, params)
+  return ret
+
+
+def print_sizes(file, params):
+  if not os.path.isfile(file):
+    return error('Input file %s not foune' % file)
+  ftype, fencoding = mimetypes.guess_type(file)
+  if (ftype == 'application/wasm' or ftype == 'text/vnd.a'):
+    js_size = 0
+    wasm_file = file
+  else:
+    if '.' in file:
+      file, extension = file.rsplit('.', 1)
+
+    # Find the JS file size
+    st = os.stat(file)
+    js_size = st.st_size
+
+    # Find the rest of the sizes
+    wasm_file = file + '.wasm'
 
-
-def print_sizes(js_file):
-  if not os.path.isfile(js_file):
-    return error('Input JS file %s not foune' % js_file)
-  if not js_file.endswith('.js'):
-    return error('Input file %s does not have a JS extension' % js_file)
-
-  basename = js_file[:-3]
-
-  # Find the JS file size
-  st = os.stat(js_file)
-  js_size = st.st_size
-
-  # Find the rest of the sizes
-  wasm_file = basename + '.wasm'
   if not os.path.isfile(wasm_file):
     return error('Wasm file %s not found' % wasm_file)
 
-  sizes = shared.check_call([LLVM_SIZE, '--format=sysv', wasm_file],
-                            stdout=subprocess.PIPE).stdout
+  callArgs = [LLVM_SIZE] + params + [wasm_file]
+  sizes = shared.check_call(callArgs, stdout=subprocess.PIPE).stdout
```

Comment:
How about `cmd` or `call_args` rather than `callArgs`?   But also maybe just inline it and avoid the new var completely?

Reply:
Sounds both good to me. My preference would be call_args.